### PR TITLE
Contacts API not working on Android ICS

### PR DESCRIPTION
**Pull request by michelemorello**
Jira issue: WP-786

### CODE REVIEW

File: webinos/platform/android/impl/src/org/webinos/impl/ContactManagerImpl.java [hunk 1]
```diff
@@ -373,35 +377,41 @@ private int printDB(Cursor cursor){
...
+	    		contact.note = getContactNote(cursor, contactID);
+	    		contact.photos = getContactPhotos(contactID);					//to be tested
+	    		contact.categories = getContactCategories(cursor, contactID);	//to be tested
+	    		contact.urls = getContactUrls(cursor, contactID);
+	    		contact.gender = null;											//not implemented on android
+	    		contact.timezone = null;										//not implemented on android
+	    		contacts.add(contact);
+	        }
+	    }
+	    Log.v("DBG", "000");
```
Comment (andreasbotsikas) [inline webinos/platform/android/impl/src/org/webinos/impl/ContactManagerImpl.java]: Can you comment out this log message?

**Comment by jplyle** [comment]
Please: do not merge until WP-718 is merged.

**Comment by paolovergori** [comment]
Sorry, but I don't understand the relation with WP-718. Why we should wait for it to be merged?

**Comment by jplyle** [comment]
Sorry - ignore me.  We were having some irritating testing problems and I was _desperate_ to get WP-718 merged without more conflicts.  Please go ahead.

**Comment by jplyle** [comment]
Any chance of a fix for http://jira.webinos.org/browse/WP-811 as part of this?

**Comment by andreasbotsikas** [comment]
@michelemorello is this still relevant? Do we need this merged for living labs too? @jplyle do the tests still fail or was it fixed from another PR?

**Comment by michelemorello** [comment]
Yes, it is still relevant. If you need to use contacts API on android you definitely need to merge this. It's 2 month that is pending for approval...

**Pull request merged**